### ISSUE TITLE
CompleteResponse wrong transactionId

### ISSUE THREAD

**Issue by dercoder** [hiqdev/omnipay-interkassa#1]
I think you should return in
https://github.com/hiqdev/omnipay-interkassa/blob/master/src/Message/CompletePurchaseResponse.php#L80
this "ik_pm_no"

$response->getTransactionReference(); // a reference generated by the payment gateway
$response->getTransactionId(); // the reference set by the originating website if available.

**Comment by dercoder** [comment]
And transactionReference should be "ik_inv_id"

**Comment by SilverFire** [comment]
`$response->getTransactionId()` should return `ik_pm_no`, agree on this.

> transactionReference should be "ik_inv_id"

I don't think so. Original method documentation in OmniPay is 

> reference provided by the gateway to represent this **transaction**

According to [InterKassa API doc](https://www.interkassa.com/files/docs/IK2.SCI.Protocol.v1.0.0.en.pdf), `ik_inv_id` is **invoice** identifier. Currently used `ik_trn_id` is **transaction** identifier.

**Comment by dercoder** [comment]
Ok, but `ik_inv_id` will be provided by payment gateway, not from originating website.
I also get empty `ik_trn_id` so my references are always empty.

**Comment by SilverFire** [comment]
As far as I remember, both `ik_inv_id` and `ik_trn_id` are provided by gateway, aren't they?

**Comment by dercoder** [comment]
Yes, but i think. `ik_trn_id` will be used in "Purse/Old" Mode and `ik_inv_id` in "Checkout/New" Mode.

```
  'ik_co_id' => '56b88bfe3c1eaf60555b4568',
  'ik_co_prs_id' => 'XXX',
  'ik_inv_id' => '4613462324',
  'ik_inv_st' => 'success',
  'ik_inv_crt' => '2016-03-10 14:44:52',
  'ik_inv_prc' => '2016-03-10 14:44:52',
  'ik_trn_id' => '',
  'ik_pm_no' => '132',
  'ik_pw_via' => 'test_interkassa_test_xts',
  'ik_am' => '70.84',
  'ik_co_rfn' => '70.84',
  'ik_ps_price' => '72.97',
  'ik_cur' => 'RUB',
  'ik_desc' => 'Deposit: 132',
  'ik_sign' => 'NH7nhXcTZ5vXXXXXKZhL16QzcC0DIwkh/RqoM=',
```

**Comment by SilverFire** [comment]
Let me check it

**Comment by SilverFire** [comment]
https://github.com/hiqdev/omnipay-interkassa/commit/6731ed2c04eb4cd2bf8ef7a643fb21bf00c88a96
https://github.com/hiqdev/omnipay-interkassa/commit/de874bd4438b1f4755c7444f7fb158f324e8efbb
https://github.com/hiqdev/omnipay-interkassa/commit/058bbc5dbd62e7832bebf0571d7150484eeb410b

**Comment by SilverFire** [comment]
> Yes, but i think. ik_trn_id will be used in "Purse/Old" Mode and ik_inv_id in "Checkout/New" Mode.

You're right. Fixed: 1368089
